pocs/tests/test_observatory.py

```diff
@@ -36,8 +36,11 @@ def images_dir(tmpdir_factory):
 
 
 def test_error_exit(config):
+    # TODO Describe why this is expected to fail, and how it is different
+    # from the other tests, esp. test_bad_mount_port.
+    # pytest.set_trace()
     with pytest.raises(SystemExit):
-        Observatory(ignore_local_config=True, simulator=['none'])
+        Observatory(ignore_local_config=True, config=config, simulator=['none'])
```

Comment:
Is this `config` needed here?

Reply:
If config is not required here, then why does this test depend on the config fixture?

Really, I was confused during debugging to find that the config gets loaded from the file(s) even though the test depends on the config fixture.

Reply:
It can probably removed from the test. I think in normal situations we would want the config fixture to be used but this is testing that the default config file cannot load properly because there is no associated hardware.

Reply:
Or at least that's the idea. With hardware attached this PR is needed.